src/main/deploy/practiceBot.xml

```diff
@@ -1,54 +1,46 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I thought the goal of this was to eliminate this file, so why are we changing it in the deploy folder instead of deleting it?